docs/restore-individual-tables.md

```diff
@@ -44,33 +66,35 @@ The files are located in the target directory:
 
 ## Import the table
 
-Import the table by doing the following:
+Now let's move the table to its new home. To import the table, on the target server,you need to do the following:
 
-* Create a copy of the table
-* Discard the table's original tablespace
-* Copy the `.ibd` and `.cfg` files to the location
-* Import the tablespace
+* Use the schema and create an empty copy of the table in the new location.
```

Comment:
we should explain how to use use .SQL files saved from our script.

1. mysql -uroot < ./schema_dumps/test2/create_database_test2.sql . This statement creates a database only if it doesn't exist. 

2.  CREATE the desired table: mysql -uroot < ./schema_dumps/test2/t1.sql. If the table already exists, the above statement will fail. "ERROR 1050 (42S01) at line 34: Table 't1' already exists " It is users responsibility to drop the conflicting table and retry the operation

3. Ensure the table the created mysql -uroot -N -e"SHOW TABLES" test2
+----+
| t1 |
+----+

4.  Now the next ALTER TABLE test.t1 DISCARD TABLESPACE

I guess, we should this name "export_test" everywhere for consistency purpose?
| t4 |
+----+

Reply:
or simply say "If you used the script (dump-schema.md) during the backup, you have saved the table definitions. You may use the appropriate table .sql file to create the table."

I guess this one line adds more value:
mysql -uroot < ./schema_dumps/test2/create_database_test2.sql

Reply:
The entire point of the script is to use it for restore. Let's elaborate on it a bit more. Can you please refine this text and add it?

"

or simply say "If you used the script (dump-schema.md) during the backup, you have saved the table definitions. You may use the appropriate table .sql file to create the table."

I guess this one line adds more value:
mysql -uroot < ./schema_dumps/test2/create_database_test2.sql
"